Libplanet/Blocks/BlockHeader.cs

```diff
@@ -200,6 +211,14 @@ internal void Validate(DateTimeOffset currentTime)
                 );
             }
 
+            if (Difficulty > TotalDifficulty)
+            {
+                throw new InvalidBlockTotalDifficultyException(
+                    "total difficulty must be less than its difficulty, " +
```

Comment:
In my thought, total difficulty will be always same or bigger than its difficulty. doesn't it?

| Block Index | Difficulty | Total Difficulty |
| -------------:|-------------:| -----:|
| 0     | 0 | 0 |
| 1 | 1000 | 1000 |
| 2 | 1100 | 2100 |